myhpom/views/signup.py

```diff
@@ -24,7 +24,7 @@ def signup(request):
             user = authenticate(username=form.data['email'], password=form.data['password'])
             login(request, user)
 
-            if user_details.state.advance_directive_template:
+            if len(user_details.state.healthnetwork_set.all()) > 0:
```

Comment:
https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.exists should be used here. (not important for today but in general should always use exists vs len of the query set which can be extremely inefficient way to test for existence)